packages/main/src/backend/addressbook.ts

```diff
@@ -0,0 +1,192 @@
+import type { AxiosError } from 'axios';
+import type { AddressbookEntry, Channels, ErrorResult } from '../../../types';
+import Backend from './backendInstance';
+import { printAxiosError } from './backendInstance';
+
+const endpoint ='/address-book/entries';
+const channelsEndpointExtension = 'channels';
+
+export async function createAddressbookEntry(entry:AddressbookEntry):Promise<ErrorResult<AddressbookEntry>> {
+    try{
+        
+        const response = await Backend.instance.post(`${endpoint}`, {...entry, id: undefined});
+        return {res: response.data, error:false};
```

Comment:
are you sure the server returns a boolean not just an HTTP status code indicating what happend?